pages/changelog.rst

```diff
@@ -2,6 +2,91 @@
+- Support custom chart legends and value actions in widgets. `Graylog2/graylog2-server#6612 <https://github.com/Graylog2/graylog2-server/issues/6612>`_ `Graylog2/graylog2-server#7207 <https://github.com/Graylog2/graylog2-server/issues/7207>`_ `Graylog2/graylog2-server#9844 <https://github.com/Graylog2/graylog2-server/issues/9844>`_
+- Allow highlighting of dates and boolean values. `Graylog2/graylog2-server#9835 <https://github.com/Graylog2/graylog2-server/issues/9835>`_ `Graylog2/graylog2-server#10000 <https://github.com/Graylog2/graylog2-server/issues/10000>`_
+- Add static/gradient color options to value highlighting. `Graylog2/graylog2-server#10033 <https://github.com/Graylog2/graylog2-server/issues/10033>`_
+- Support for connecting to newer Kafka clusters. `Graylog2/graylog2-server#7101 <https://github.com/Graylog2/graylog2-server/issues/7101>`_ `Graylog2/graylog2-server#7783 <https://github.com/Graylog2/graylog2-server/issues/7783>`_ `Graylog2/graylog2-server#9927 <https://github.com/Graylog2/graylog2-server/issues/9927>`_ `Graylog2/graylog2-server#10146 <https://github.com/Graylog2/graylog2-server/issues/10146>`_
+- Support IPinfo databases in GeoIP lookup data adapter. `Graylog2/graylog2-server#10183 <https://github.com/Graylog2/graylog2-server/issues/10183>`_
+- Add ``elasticsearch_mute_deprecation_warnings`` config option to disable Elasticsearch related deprecation warnings. `Graylog2/graylog2-server#9822 <https://github.com/Graylog2/graylog2-server/issues/9822>`_ `Graylog2/graylog2-server#10163 <https://github.com/Graylog2/graylog2-server/issues/10163>`_ `Graylog2/graylog2-server#10239 <https://github.com/Graylog2/graylog2-server/issues/10239>`_
+- Vastly improved search aggregation builder. `Graylog2/graylog2-server#9742 <https://github.com/Graylog2/graylog2-server/issues/9742>`_ `Graylog2/graylog2-server#10212 <https://github.com/Graylog2/graylog2-server/issues/10212>`_ `Graylog2/graylog2-server#10281 <https://github.com/Graylog2/graylog2-server/issues/10281>`_ `Graylog2/graylog2-server#10293 <https://github.com/Graylog2/graylog2-server/issues/10293>`_ `Graylog2/graylog2-server#10301 <https://github.com/Graylog2/graylog2-server/issues/10301>`_
+- Add ``allowed_auxiliary_paths`` config option to optionally restrict file system access for the CSV File data dapter. `Graylog2/graylog2-server#10244 <https://github.com/Graylog2/graylog2-server/issues/10244>`_
+- Add timerange indicator to widgets. `Graylog2/graylog2-server#7358 <https://github.com/Graylog2/graylog2-server/issues/7358>`_ `Graylog2/graylog2-server#10508 <https://github.com/Graylog2/graylog2-server/issues/10508>`_
+- Allow filtering for entity ID in pagination. `Graylog2/graylog2-server#10537 <https://github.com/Graylog2/graylog2-server/issues/10537>`_
+- Add built-in Prometheus exporter. `Graylog2/graylog2-server#10767 <https://github.com/Graylog2/graylog2-server/issues/10767>`_
```

Comment:
We can also mention https://github.com/Graylog2/graylog2-server/pull/9836 here. The title could be: "Add color scale configuration options for heatmap widget."

Reply:
Done, thanks!